docker/bigtop-puppet/build.sh

```diff
@@ -63,5 +63,10 @@ case "${OS}-${VERSION}" in
     ;;
 esac
 
+if [ $OS == "openeuler" ];then
+  OPENEULER_OS="${OS}/${OS}"
+  sed -i -e "s|${OS}:${VERSION}|$OPENEULER_OS:${VERSION}|" ./Dockerfile
+fi
+
```

Comment:
Suggest to use the 'case'/'esac' logic, as set by in L54~L64. Like this:
```
  openeuler-*)
  OPENEULER_OS="${OS}/${OS}"
      sed -i -e "s|${OS}:${VERSION}|$OPENEULER_OS:${VERSION}|" ./Dockerfile
    ;;

```

Reply:
tks, it's done. 
[67721e5](https://github.com/apache/bigtop/pull/1051/commits/67721e505533cb54a023aa5f2a3828966daff52d)